packages/webview_flutter/webview_flutter_android/android/src/main/java/io/flutter/plugins/webviewflutter/WebViewHostApiImpl.java

```diff
@@ -156,6 +156,13 @@ public WebChromeClient getWebChromeClient() {
       return currentWebChromeClient;
     }
 
+    @Override
+    protected void onScrollChanged(int l, int t, int oldL, int oldT) {
+      super.onScrollChanged(l, t, oldL, oldT);
+      api.onScrollChanged(
+          this, (long) l, (long) t, reply -> {});
```

Comment:
Is there a reason we're not mirroring the full API here, and sending all of the same arguments?

Reply:
@stuartmorgan Yes, at first I mirrored the full API but then @bparrishMines realized that the equivalent API on the iOS side do not provide `oldL` and `oldT`. That is why we decided to leave those out to make the APIs identical for both iOS and Android. Besides, users could implement their own logic to store the previous `left`, `top` so I want to keep this.

Reply:
That's an argument for omitting them in the cross-platform API, but what's the argument for not mirroring it in the low-level Dart API that's intended to directly expose/translate the Android APIs?

Reply:
> That's an argument for omitting them in the cross-platform API, but what's the argument for not mirroring it in the low-level Dart API that's intended to directly expose/translate the Android APIs?

Because I don't see why we should mirror parameters that we are not gonna use at cross-platform API. If it is a convention to fully mirror the native APIs then I am happy to follow it. Do you want me to change it?

Reply:
Yes, the convention in this package is that we directly mirror the platform APIs when possible.